tests/shared/core/test_domain.py

```diff
@@ -1400,24 +1409,6 @@ def test_ignored_intents_slot_mappings_invalid_domain():
         Domain.from_dict(domain_as_dict)
 
 
-def test_form_with_no_required_slots_keyword():
```

Comment:
I think this test should stay, unless you enforce `required_slots` in the schema.

Reply:
re-added